h1b/src/usb/mod.rs

```diff
@@ -61,7 +124,7 @@ enum USBState {
     WaitingForSetupPacket,   // Waiting for message from host
     DataStageIn,             // Sending data to host
     NoDataStage,             // Sending status (not data) to host,
-                             // e.g. in response to set command
+    // e.g. in response to set command
```

Comment:
Line 127: was this an accidental change? It looks like you un-aligned a previously-aligned comment.

Reply:
Nope -- fixed!